RxSwiftWorkshop/Maps/Model/GeolocatorSpec.swift

```diff
@@ -34,47 +33,89 @@ class GeolocatorSpec: QuickSpec {
                 sut = nil
             }
 
-            it("should pass geolocation request to its SystemGeolocator") {
-                guard case let (disposeBag?, systemGeolocator?) = unwrap(disposeBag, systemGeolocator) else { return }
+            condition("always") {
                 let address = "Valencia Airport"
-                sut?.geolocate(address: address).subscribe().disposed(by: disposeBag)
-                systemGeolocator.verifyCall(
-                    withIdentifier: FakeSystemGeolocator.Identifiers.geocodeAddressString, arguments: [address]
-                )
+                setup {
+                    sut?.geolocate(address: address).subscribe().disposed(by: disposeBag!)
```

Comment:
You won't have to remember that – compiler will tell you. Also, the reason why I prefer avoiding the IUOs is that they are stating the possibility of crash only in the variable declaration, not in the actual place of crashing, which is the variable use. Dropping IUO and bringing force unwrapping to the place of use brings the expression of "here we could crash" back to the actual place of crashing. If it's not a deal breaker for you,  I'm gonna keep wringing this way.